### DISCUSSION TITLE
MySQL and PostgreSQL Compatible Databases

### DISCUSSION THREAD

**Discussion by topefolorunso** [mindsdb/mindsdb#5325] [General]
Hi @ZoranPandovski . As per your review here
https://github.com/mindsdb/mindsdb/pull/5259#pullrequestreview-1363007227, I have discovered a few databases that are forks of MySQL and PostgreSQL below.

# MySQL
- [Percona Server](https://github.com/percona/percona-server)
- [go-mysql-server](https://github.com/dolthub/go-mysql-server) 
- [PolarDB (Alibaba)](https://www.alibabacloud.com/product/polardb)

# PostgreSQL 
- [AlloyDB (Google)](https://cloud.google.com/alloydb)
- [PolarDB (Alibaba)](https://www.alibabacloud.com/product/polardb)
- [TBase (Tencent)](https://github.com/Tencent/TBase)
- [Citus  (Microsoft)](https://www.citusdata.com/about/our-story/)
- [Greenplum (VMWare)](https://greenplum.org/)

_P.S: I'm still searching for more_

**Comment by federico-razzoli** [comment]
I suppose you're only considering open source DBMSs that are not yet supported by MindsDB, correct?

I think you should include: 

- [Galera](https://galeracluster.com/downloads/#downloads) is a high availability solution built-in MariaDB, but the vendor also provides MySQL builds with Galera includes.
- [TiDB](https://www.pingcap.com/), which is not a fork but it's mostly compatible with MySQL.
- [Vitess](https://vitess.io/) is also mostly MySQL-compatible and it's built to implement data sharding.
